.github/workflows/deployment.yml

```diff
@@ -11,7 +11,7 @@ on:
     inputs:
       version:
         description: Version to be published
-        default: v1.18.0
+        default: v1.18.1
```

Comment:
We might need to wait for the doc and the version change before merging this to `next`